Emby.Server.Implementations/Library/DefaultAuthenticationProvider.cs

```diff
@@ -70,9 +70,9 @@ public Task<ProviderAuthenticationResult> Authenticate(string username, string p
                 byte[] calculatedHash = _cryptographyProvider.ComputeHash(
                     readyHash.Id,
                     passwordbytes,
-                    readyHash.Salt);
+                    readyHash.Salt.ToArray());
 
-                if (calculatedHash.SequenceEqual(readyHash.Hash))
+                if (readyHash.Hash.SequenceEqual(calculatedHash))
```

Comment:
why change?.. I don't see much difference, looks a bit pointless

Reply:
TBH, I have no idea

Reply:
Can we revert it then? Change because change isn't a good thing to have IMHO

Reply:
It looks the same to me.

Reply:
It's probably checking the same thing, so I think we shouldn't change the line "just because we can" - it complicates future investigations/bisections/etc.

Reply:
Changing it back now won’t fix that

Reply:
It would if you rebase and edit the commit that touched that line :)

Reply:
Ok, so now I can rebase :p